### ISSUE TITLE
Export definition of different dictionaries in different fields

### ISSUE THREAD

**Issue by ddc251992** [FooSoft/yomichan#1345]
1) I was trying to modify the handlebars from yomichan to see if I could export the definitions available from the dictionaries in different fields of my card in anki but I don't seem to get it right. 
2) Also I don't know if it is possible to specify, how many definition could be imported from each dictionary.

Could it be possible to have a sample code explaining how to do it (if it is possible)?
Thanks for your time.
![1](https://user-images.githubusercontent.com/78541446/106891186-48812f80-66e2-11eb-9097-cacb46b53bcb.jpg)

**Comment by toasted-nutbread** [comment]
Here is an example based off the latest definitions. Note that the `{glossary}` and `{glossary-single}` definitions have changed since the last release, but the behaviour is the same.

Base file:
https://github.com/FooSoft/yomichan/blob/6c9f4a2be98012e791485dadb7078c16cb8d063c/ext/bg/data/default-anki-field-templates.handlebars

Modifications patch below. If you are unfamiliar with diffs, the red is the value of the old file, the green is the changes or additions you would have to make.

```patch
diff --git a/ext/bg/data/default-anki-field-templates.handlebars b/ext/bg/data/default-anki-field-templates.handlebars
index 0f390e8d7..ab2f97573 100644
--- a/ext/bg/data/default-anki-field-templates.handlebars
+++ b/ext/bg/data/default-anki-field-templates.handlebars
@@ -1,4 +1,5 @@
 {{#*inline "glossary-single"}}
+    {{~#if (op "||" (op "===" targetDictionary undefined) (op "===" targetDictionary dictionary))~}}
     {{~#unless brief~}}
         {{~#scope~}}
             {{~#set "any" false}}{{/set~}}
@@ -28,6 +29,7 @@
         <ul>{{#each glossary}}<li>{{#multiLine}}{{.}}{{/multiLine}}</li>{{/each}}</ul>
     {{~/if~}}
     {{~#set "previousDictionary" dictionary~}}{{~/set~}}
+    {{~/if~}}
 {{/inline}}
 
 {{#*inline "audio"}}
@@ -101,12 +103,12 @@
     <div style="text-align: left;">
     {{~#scope~}}
         {{~#if (op "===" definition.type "term")~}}
-            {{~> glossary-single definition brief=brief noDictionaryTag=noDictionaryTag ~}}
+            {{~> glossary-single definition brief=brief noDictionaryTag=noDictionaryTag targetDictionary=targetDictionary ~}}
         {{~else if (op "||" (op "===" definition.type "termGrouped") (op "===" definition.type "termMerged"))~}}
             {{~#if (op ">" definition.definitions.length 1)~}}
-                <ol>{{~#each definition.definitions~}}<li>{{~> glossary-single . brief=../brief noDictionaryTag=../noDictionaryTag ~}}</li>{{~/each~}}</ol>
+                <ol>{{~#each definition.definitions~}}<li>{{~> glossary-single . brief=../brief noDictionaryTag=../noDictionaryTag targetDictionary=../targetDictionary ~}}</li>{{~/each~}}</ol>
             {{~else~}}
-                {{~#each definition.definitions~}}{{~> glossary-single . brief=../brief noDictionaryTag=../noDictionaryTag ~}}{{~/each~}}
+                {{~#each definition.definitions~}}{{~> glossary-single . brief=../brief noDictionaryTag=../noDictionaryTag targetDictionary=../targetDictionary ~}}{{~/each~}}
             {{~/if~}}
         {{~else if (op "===" definition.type "kanji")~}}
             {{~#if (op ">" definition.glossary.length 1)~}}
@@ -127,6 +129,10 @@
     {{~> glossary brief=true ~}}
 {{/inline}}
 
+{{~#*inline "glossary-jmdict"~}}
+    {{~> glossary targetDictionary="JMdict (English)" ~}}
+{{~/inline~}}
+
 {{#*inline "kunyomi"}}
     {{~#each definition.kunyomi}}{{.}}{{#unless @last}}, {{/unless}}{{/each~}}
 {{/inline}}
```

In short:
* An extra paramater `targetDictionary` is passed to `glossary-single`, which filters out the unwanted dictionaries.
* The invocations of `glossary-single` are updated to pass the value of `targetDictionary`.
* A new inline partial definition is added for `glossary-jmdict`. Repeat this pattern for your other dictionaries.

**Comment by ddc251992** [comment]
Thank you so much solved the problem. I also leave here the example I used for anyone interested.

![template](https://user-images.githubusercontent.com/78541446/106976536-7dc46680-6750-11eb-9bfc-53d9c13bf7d1.jpg)
[Dictionary HandleBars_without bullet points.txt](https://github.com/FooSoft/yomichan/files/5929482/Dictionary.HandleBars_without.bullet.points.txt)
